src/preset_cli/api/clients/dbt.py

```diff
@@ -514,53 +87,54 @@ class JobSchema(PostelSchema):
     """
 
     id = fields.Integer(allow_none=True)
-    account_id = fields.Integer()
-    project_id = fields.Integer()
-    environment_id = fields.Integer()
     name = fields.String()
-    dbt_version = fields.String(allow_none=True)
-    raw_dbt_version = fields.String(allow_none=True)
-    triggers = fields.Nested(TriggerSchema)
-    execute_steps = fields.List(fields.String())
-    settings = fields.Nested(SettingsSchema)
-    state = fields.Integer()
-    generate_docs = fields.Boolean()
-    schedule = fields.Nested(ScheduleSchema)
-
-    # not present in the spec
-    lifecycle_webhooks_url = fields.String(allow_none=True)
-    cron_humanized = fields.String()
-    created_at = fields.DateTime()
-    next_run = fields.DateTime(allow_none=True)
-    lifecycle_webhooks = fields.Boolean(allow_none=True)
-    next_run_humanized = fields.String(allow_none=True)
-    deferring_job_definition_id = fields.Integer(allow_none=True)
-    deactivated = fields.Boolean()
-    is_deferrable = fields.Boolean()
-    updated_at = fields.DateTime()
-    execution = fields.Nested(ExecutionSchema)
-    run_failure_count = fields.Integer()
-    run_generate_sources = fields.Boolean()
-    generate_sources = fields.Boolean()
 
 
 class ModelSchema(PostelSchema):
     """
     Schema for a model.
     """
 
-    depends_on = fields.List(fields.String(), data_key="dependsOn")
-    children = fields.List(fields.String(), data_key="childrenL1")
+    depends_on = fields.List(fields.String())
+    children = fields.List(fields.String())
     database = fields.String()
     schema = fields.String()
     description = fields.String()
     meta = fields.Raw()
     name = fields.String()
-    unique_id = fields.String(data_key="uniqueId")
+    alias = fields.String(allow_none=True)
+    unique_id = fields.String()
     tags = fields.List(fields.String())
     columns = fields.Raw(allow_none=True)
     config = fields.Dict(fields.String(), fields.Raw(allow_none=True))
 
+    @pre_load
+    def rename_fields(  # pylint: disable=unused-argument
+        self,
+        data: Dict[str, Any],
+        **kwargs: Any,
+    ) -> Dict[str, Any]:
+        """
+        Handle keys that can have camelCase or snake_case and Core/Cloud differences.
+        """
+        if "uniqueId" in data:
+            data["unique_id"] = data.pop("uniqueId")
+
+        if "childrenL1" in data:
+            data["children"] = data.pop("childrenL1")
+
+        if isinstance(columns := data.get("columns"), dict):
+            data["columns"] = list(columns.values())
+
+        if "dependsOn" in data:
+            data["depends_on"] = data.pop("dependsOn")
+        depends_on = data.get("depends_on", [])
+
+        if isinstance(depends_on, dict):
+            data["depends_on"] = depends_on["nodes"]
+
```

Comment:
Do we need this for `ModelSchema` too? I see down there that this logic was implemented in `get_og_metric_from_config` (before loading the metric to `OGMetricSchema`) but curious if we also need to do this for model.

Reply:
I think so, this is from the manifest:

```python
  "nodes": {
    "model.superset_examples.messages_channels": {
      "raw_sql": "...",
      "compiled": true,
      "resource_type": "model",
      "depends_on": {
        "macros": [],
        "nodes": [
          "source.superset_examples.public.channels",
          "source.superset_examples.public.messages"
        ]
      },
```

Reply:
Gotcha! Thanks for confirming it